images/win/scripts/Tests/WinAppDriver.Tests.ps1

```diff
@@ -0,0 +1,19 @@
+Describe "WinAppDriver" {
+  It "WinAppDriver directory exists" {
+    $wad = "Windows Application Driver"
+    if (${env:ProgramFiles(x86)})
```

Comment:
> The code inherited from the Validation script. I guess this 2016/2019 branching. To remove it there must be additional investigation

Yes, please do it the additional investigation and remove the redundancy code.

Reply:
I see no sense to do this check for 32/64 architecture and ensured both 2016 and 2019 images have Windows Application Driver in the same location. So the branch is removed